### PR TITLE
Fix the seed of a flaky `test_sampling` test

### PR DESCRIPTION
**Pull request by aalexandrov**
*Issue #, if available:*

*Description of changes:*

Fixes the seed of the `test_sampling` which otherwise might be flaky.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by szha** [comment]
Job PR-129/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-129/1/index.html

File: test/distribution/test_distribution_sampling.py [hunk 1]
```diff
@@ -67,6 +67,7 @@
 DISTRIBUTIONS_WITH_CDF = [Gaussian, Uniform, Laplace, Binned]
 
 
+@pytest.mark.seed(1960264129)  # fix a static seed for a flaky test
```
Comment (jaheba) [inline test/distribution/test_distribution_sampling.py]: Where does this magic number come from?

**Comment by jaheba** [comment]
I think this particular issue was fixed in #120.

The issue here was that `log(0)` was run in some cases which evaluates to `-inf` in `mx.nd`.

**Comment by aalexandrov** [comment]
You are right, the issue only occurs in versions up to 0.1.4

**Pull request closed without merging**